modules/analysis/src/openapi.rs

```diff
@@ -0,0 +1,22 @@
+use utoipa::openapi::{Object, RefOr, Schema, SchemaType};
+use utoipa::OpenApi;
+
+#[derive(OpenApi)]
+#[openapi(paths(), components(), tags())]
+pub struct ApiDoc;
+
+pub fn openapi() -> utoipa::openapi::OpenApi {
+    let mut doc = ApiDoc::openapi();
+
+    doc.merge(endpoints::ApiDoc::openapi());
+
+    if let Some(components) = doc.components.as_mut() {
```

Comment:
Is this required at this level, particularly as UUIDs are pretty spread out, maybe we do this higher-up where we merge all of the openapi's together?